msobooking.yaml

```
@@ -0,0 +1,436 @@
+      ol.releasedatetime:
+        type: "ol.releasedatetime"
+        column: "RELDATE"
+        transforms:
+        - !<transforms.DateAsDateTimeTransform>
+          pattern: ["yyyy-MM-dd"]
+          timezone: "America/New_York"
+      criminaljustice.releasetype:
+        type: "criminaljustice.releasetype"
+        column: "RELTYPE"
+      ol.admissiontype:
+        type: "ol.admissiontype"
+        column: "ADTYPE"
+      ol.commitdatetime:
+        type: "ol.commitdatetime"
+        column: "COMDATE"
+        transforms:
+        - !<transforms.DateAsDateTimeTransform>
+          pattern: ["yyyy-MM-dd"]
+          timezone: "America/New_York"
+    name: "jailstay"
+
+  person:
+    fqn: "general.person"
+    entitySetName: "MSOPerson"
+    propertyDefinitions:
+      nc.SubjectIdentification:
+        type: "nc.SubjectIdentification"
+        column: "PCP"
+      nc.PersonGivenName:
+        type: "nc.PersonGivenName"
+        column: "FIRSTNAM"
+      nc.PersonSurName:
+        type: "nc.PersonSurName"
+        column: "LASTNAME"
+      nc.SSN:
+        type: "nc.SSN"
+        column: "SSNUM"
+      nc.PersonSex:
+        type: "nc.PersonSex"
+        column: "SEX"
+      nc.PersonSuffix:
+        type: "nc.PersonSuffix"
+        column: "AFFIX"
+      nc.PersonBirthDate:
+        type: "nc.PersonBirthDate"
+        column: "BIRTH"
+        transforms:
+        - !<transforms.DateTransform>
+          pattern: ["yyyy-MM-dd"]
+          timezone: "America/New_York"
+      nc.PersonEthnicity:
+        type: "nc.PersonEthnicity"
+        column: "ETHNICITY"
+      nc.PersonRace:
+        type: "nc.PersonRace"
+        column: "RACE"
+      nc.PersonMiddleName:
+        type: "nc.PersonMiddleName"
+        column: "MIDDLE"
+      ol.idnumber:
+        type: "ol.idnumber"
+        column: "FBI"
+    name: "person"
+
+  temporaryrelease:
+    fqn: "ol.temporaryrelease"
+    entitySetName: "MSOTemporaryrelease"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "RELSTDT"
+      ol.datetimestart:
+        type: "ol.datetimestart"
+        column: "RELSTDT"
+        transforms:
+        - !<transforms.DateAsDateTimeTransform>
+          pattern: ["yyyy-MM-dd"]
+          timezone: "America/New_York"
+      ol.datetimeend:
+        type: "ol.datetimeend"
+        column: "RELRNDT"
+        transforms:
+        - !<transforms.DateAsDateTimeTransform>
+          pattern: ["yyyy-MM-dd"]
+          timezone: "America/New_York"
+      ol.projectedenddatetime:
+        type: "ol.projectedenddatetime"
+        column: "RELDBDATE"
+    name: "temporaryrelease"
+
+  releaseofficer:
+    fqn: "ol.employee"
+    entitySetName: "MSOOfficer"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "RELOFF"
+      publicsafety.employeeid:
+        type: "publicsafety.employeeid"
+        column: "RELOFF"
+    name: "releaseofficer"
+
+  dnatest:
+    fqn: "ol.medicaltest"
+    entitySetName: "MSODnatest"
```

```suggestion
    entitySetName: "MSODnaSample"
```
change the name of this to "sample" and not "test", remove date and put in association.